docs/quick-start.mdx

```diff
@@ -4,134 +4,339 @@ title: Quick Start
-. "$here\$sut"
+Looking at the last line of output you can see that we run 1 test and it Passed. Good job, you just run your first Pester test! 🥳🥳🥳
 
-Describe "Clean" {
-    It "does something useful" {
-        $true | Should Be $false
-    }
+### Understanding our test
+
+In the previous run, our test passed, and if you'd run it again it would pass again. That is the beauty of automated testing. This is because using the `Should` keyword we are saying:
+
+> 👉 "There should be 8 items in `$allPlanets`."
+
+And there are. 
+
+But how did we know that we want to test for exactly that? Well, we didn't. It was just one example of how we could describe our Solar System. You can try remembering some facts about it and try writing them as tests.
+
+Here are few examples: 
+
+- Earth is the third planet in our Solar System. 
+- Pluto is not part of our Solar System.
+- The planets go in this order: Mercury, Venus, Earth, Mars, Jupiter, Saturn, Uranus, Neptune.
+  
+
+```powershell
+It 'Earth is the third planet in our Solar System' {
+    $allPlanets = Get-Planet
+    $allPlanets[2].Name | Should -Be 'Earth'
 }
 ```
 
-With this skeleton of a function called "Clean" and a failing test, Test-Driven Development activities can start.
+```powershell
+It 'Pluto is not part of our Solar System' {
+    $allPlanets = Get-Planet
+    $plutos = $allPlanets | Where-Object Name -EQ 'Pluto'
+    $plutos.Count | Should -Be 0
+}
+```
+
+```powershell
+It 'Planets have this order: Mercury, Venus, Earth, Mars, Jupiter, Saturn, Uranus, Neptune' {
+    $allPlanets = Get-Planet
+    $planetsInOrder = $allPlanets.Name -join ', ' 
+    $planetsInOrder | Should -Be 'Mercury, Venus, Earth, Mars, Jupiter, Saturn, Uranus, Neptune'
+} 
+```
+
+Try adding those tests into your `Get-Planet.Tests.ps1` file. Put them under the other `It` block, but make sure they are placed within the curly braces associated with `Describe`.
 
-Pester test script filenames _must_ end with `.Tests.ps1` suffix in order for [Invoke-Pester](./commands/Invoke-Pester)
-to discover and run them. (eg. MyScript.Tests.ps1).
+### Breaking our test, by breaking the tested function
 
-Test scripts named with or without this suffix can be run manually using -Script parameter.
+There are few ways to break the test, one of them is adding Pluto back into our Solar System.
 
 ```powershell
-Invoke-Pester -Script .\Test-ScriptWithPesterTest.ps1
+$planets = @(
+    @{ Name = 'Mercury' }
+    @{ Name = 'Venus'   }
+    @{ Name = 'Earth'   }
+    @{ Name = 'Mars'    }
+    @{ Name = 'Jupiter' }
+    @{ Name = 'Saturn'  }
+    @{ Name = 'Uranus'  }
+    @{ Name = 'Neptune' }
+    @{ Name = 'Pluto'   }
+) | ForEach-Object { [PSCustomObject] $_ }
+```
+
+This will break the assertion that we have in our test, because we no longer return 8 items from the tested function. Instead we now return 9. Running the test, it will no longer pass: 
+
+```shell
+Invoke-Pester -Output Detailed C:\t\Planets\Get-Planet.Tests.ps1
```

Comment:
Same here

Reply:
Usually there is no need to spell out every occurrence, if I fix it in one place I will fix it in all. Just to make it a bit easier for you :)